src/components/Tabs/Annotation/AnnotationImportButton.tsx

```diff
@@ -16,6 +17,19 @@ type AnnotationImportButtonProps = {
   annotationState: AnnotationState;
 };
 
+const MultilineRadio = styled(Radio)<{ $expanded?: boolean }>`
+  & .ant-radio.ant-wave-target {
+    ${(props) => {
+      if (props.$expanded) {
+        return css`
+          margin: 3px 0 auto 0;
+        `;
+      }
+      return css``;
+    }}
+  }
+`;
```

Comment:
![image](https://github.com/user-attachments/assets/79657ad3-81e8-4953-8609-e423ac503404)

Used for the "Merge matching annotations" radio button, to make sure that the alignment looks correct when it's expanded.

Reply:
When not selected/collapsed:
![image](https://github.com/user-attachments/assets/b77e80b0-9b1b-4668-88f8-10ede5f6a6e9)